nipype/interfaces/afni/utils.py

```diff
@@ -277,7 +267,7 @@ class BrickStatInputSpec(CommandLineInputSpec):
         traits.Float,
         traits.Float,
         desc="p0 ps p1 write the percentile values starting "
-        "at p0% and ending at p1% at a step of ps%. "
+        "at p0\\% and ending at p1\\% at a step of ps%. "
```

Comment:
Why do these need to be escaped? Also why not `%.`?

Reply:
Would need to check. Apparently no good reason